godot-core/src/obj/traits.rs

```diff
@@ -182,6 +182,19 @@ pub trait EngineEnum: Copy {
     }
 }
 
+/// Auto-implemented for all engine-provided bitfields.
+pub trait EngineBitfield: Copy {
+    fn try_from_ord(ord: u64) -> Option<Self>;
+
+    /// Ordinal value of the bitfield, as specified in Godot.
+    fn ord(self) -> u64;
+
+    fn from_ord(ord: u64) -> Self {
+        Self::try_from_ord(ord)
+            .unwrap_or_else(|| panic!("ordinal {ord} does not map to any bitfield"))
+    }
+}
```

Comment:
Yeah, since we are dealing with bitfields, it makes sense for it to be a combination of several possible bitfields :+1: 

Will change the wording to make sense, sorry about that! :^)

Reply:
Rephrased the sentences a bit, does this work?

```
/// Ordinal value of the bit flag, as specified in Godot.`
[...]
.unwrap_or_else(|| panic!("ordinal {ord} does not map to any valid bit flag"))
```